tests/src/FunctionalJavascript/ContactSubmissionTest.php

```diff
@@ -224,6 +224,65 @@ public function testStaticAndAutocompleteOnContactElement() {
     $this->assertEquals('Frederick-Edited', $contact_result2['values'][0]['first_name'], $result_debug);
     $this->assertEquals('Pabst-Edited', $contact_result2['values'][0]['last_name'], $result_debug);
   }
+
+  /**
+   * Test Existing Contact Element configured as Current (logged-in) User
+   */
+  public function testStaticCurrentUser() {
+    $this->drupalLogin($this->rootUser);
+    $this->drupalGet(Url::fromRoute('entity.webform.civicrm', [
+      'webform' => $this->webform->id(),
+    ]));
+    // The label has a <div> in it which can cause weird failures here.
+    $this->assertSession()->waitForText('Enable CiviCRM Processing');
+    $this->assertSession()->waitForField('nid');
+    $this->getSession()->getPage()->checkField('nid');
+    $this->assertSession()->assertWaitOnAjaxRequest();
```

Comment:
We have a helper for the above 4 lines - `$this->enableCivicrmOnWebform();`. Can we use that pls? :)